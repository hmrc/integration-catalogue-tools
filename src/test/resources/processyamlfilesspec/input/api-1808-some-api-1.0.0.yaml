openapi: 3.0.3
info:
  title: 'API#1808'
  description: >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
  version: 1.0.0
  contact:
    name: Joe Bloggs
    email: joe.bloggs@example.com

tags:
  - name: API#1808
    description: API description.
