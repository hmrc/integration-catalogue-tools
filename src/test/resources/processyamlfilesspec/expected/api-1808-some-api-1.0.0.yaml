openapi: 3.0.3
info:
  title: 'API#1808'
  description: >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
  version: 1.0.0
  contact:
    name: Joe Bloggs
    email: joe.bloggs@example.com

  x-integration-catalogue:
    reviewed-date: 2022-04-22T20:27:05Z
    platform: CORE_IF
    publisher-reference: 1808
tags:
  - name: API#1808
    description: API description.
